From 6982be353fa3b2b3f6d98a6e488688b8d7964cf4 Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Sun, 19 Aug 2018 00:48:13 -0500
Subject: [PATCH 7/7] PostLogin: NM to rule wlan on GUI

On Clear Linux systemd-networkd and NetworkManager wants to manage wlan,
so let NM rules on wlan when gdm loging session is started.

With this you don't need to disable systemd-networkd

resolve clearlinux/distribution#50
---
 data/PostLogin | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/data/PostLogin b/data/PostLogin
index 8684afe..823f5db 100755
--- a/data/PostLogin
+++ b/data/PostLogin
@@ -7,6 +7,13 @@
 # for the user or something like that.  $HOME, $LOGIN and such will all be
 # set appropriately and this script is run as root.
 
+# On Clear Linux systemd-networkd and NetworkManager wants to manage wlan,
+# so let NM rules on wlan when gdm loging session is started
+
+mkdir -p /run/systemd/network
+ln -sf /dev/null /run/systemd/network/80-wlan.network
+systemctl restart systemd-networkd
+
 if [ -e /etc/gdm/PostLogin/Default ] ; then
   source /etc/gdm/PostLogin/Default
 fi
-- 
2.18.0

